src/components/Alphabet/Alphabet.css

```diff
@@ -0,0 +1,21 @@
+.alphabet {
+    display: grid;
+    padding: 12px 20px;
+    margin: 0 auto;
+    max-width: 790px;
+    grid-template-columns: repeat(auto-fill, minmax(45px, 1fr));
+    grid-auto-rows: 40px;
+    grid-gap: 10px;
+
+    @media (min-width: 576px) {
+        grid-template-columns: repeat(auto-fill, minmax(45px, 1fr));
```

Comment:
This grid-template-columns is not necessary here (it is set for the mobile and up a few lines above)